src/index.njk

```diff
@@ -37,7 +37,7 @@ hubspot:
                 </div>
             </div>
             <div class="flex justify-center items-end m-auto lg:mx-0">
-                <lite-youtube videoid="PkuCa_wB5jU" params="rel=0" style="background-image: url('images/ff-explainer.gif'); background-size: cover; width: 500px; overflow: hidden;" class="border-4 border-indigo-200 rounded-xl sm:max-w-none sm:max-h-full lg:max-w-lg lg:max-h-72"></lite-youtube>
```

Comment:
Will the gif be removed from this repo?

Reply:
Yes, I thought I had removed it in this PR, but I inadvertently undid that change. I'll remove it.